ext/hal/st/stm32cube/stm32f1xx/soc/stm32f1xx.h

```diff
@@ -127,33 +127,33 @@
   * @{
   */
 
-#if defined(STM32F100xB)
+#if defined(STM32F100xB) || defined(STM32F100x4) || defined(STM32F100x6) || defined (STM32F100x8)

```

Comment:
Unless specific bug, which is not the case here as I can remember,
you're not supposed to modify files under /ext/hal/st/